cmd/storj-sim/network.go

```diff
@@ -177,10 +189,10 @@ func networkDestroy(flags *Flags, args []string) error {
 }
 
 // newNetwork creates a default network
-func newNetwork(flags *Flags) (*Processes, error) {
+func newNetwork(flags *Flags) (processes *Processes, cleanup func(), err error) {
```

Comment:
Adding this `cleanup` doesn't look that nice.

Reply:
where can I call cleanup instead?